src/web/db/svg_db.py

```
@@ -0,0 +1,104 @@
+                self.connection.commit()
+                return cursor.rowcount
+
+    def fetch_query(self, sql_query, params=None):
+        with self.connection.cursor() as cursor:
+            cursor.execute(sql_query, params)
+
+            result = cursor.fetchall()
+            return result
+
+
+def execute_query(sql_query: str, params: list = None):
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)